django/db/models/functions/math.py

```diff
@@ -158,9 +158,17 @@ def get_group_by_cols(self, alias=None):
         return []
 
 
-class Round(Transform):
+class Round(FixDecimalInputMixin, NumericOutputFieldMixin, Transform):
     function = 'ROUND'
     lookup_name = 'round'
+    arity = None  # Override arity=1 set by Transform to enable passing places.
```

Comment:
This is a little sneaky, but allows this to continue to be used as a transform, while enabling the option to pass `places` when using `Round()` directly.